docs/file-formats/excel2json.md

```diff
@@ -375,6 +379,11 @@ The Excel sheets must have the following structure:
       For example:
           - `list1:node1:node1.1` for the node `node1.1` in `list1`
           - `list2:node1:node1.1` for the node `node1.1` in `list2`
+- `<language>_comments`: These are optional but recommended at least for the list itself (the second row of the Excel).
+  If they are not provided for the list, the labels will be entered in the comments section.
+  Nodes will not have comments if not explicitly provided.
+  Same with the nodes, all used languages have to be entered.
```

Comment:
What means "same with the nodes"? Is there a connection between the first part of this line (before the comma), and the 2nd part of this line (after the comma)?